src/ui/display.h

```diff
@@ -15,6 +15,13 @@
 #include "../common/script.h"
 #include "../constants.h"
 
+#define MESSAGE_CHUNK_SIZE        64
+#define MESSAGE_CHUNK_PER_DISPLAY 2
+#define MESSAGE_MAX_DISPLAY_SIZE \
+    (MESSAGE_CHUNK_SIZE * MESSAGE_CHUNK_PER_DISPLAY + 2 * sizeof("...") - 1)
```

Comment:
It might be worth adding some short comment to explain what `MESSAGE_CHUNK_SIZE` and `MESSAGE_CHUNK_PER_DISPLAY`, as the first is specified in the protocol, while the second is a UI decision, and it's not too obvious by glimpsing at the code. (E.g. I initially misunderstood and modified `MESSAGE_CHUNK_SIZE` while playing with the UX... it didn't go well!).